class-two-factor-core.php

```diff
@@ -413,15 +413,37 @@ public static function get_enabled_providers_for_user( $user = null ) {
 		if ( empty( $enabled_providers ) ) {
 			$enabled_providers = array();
 		}
-		$enabled_providers = array_intersect( $enabled_providers, array_keys( $providers ) );
+		$enabled_existing_providers = array_intersect( $enabled_providers, array_keys( $providers ) );
```

Comment:
```suggestion
		$enabled_registered_providers = array_intersect( $enabled_providers, array_keys( $providers ) );
```

Would this be more clear that we are talking about enabled providers that are registered?